tests/test_iam/test_iam.py

```diff
@@ -4029,6 +4029,46 @@ def test_list_roles_none_found_returns_empty_list():
     assert len(roles) == 0
 
 
+@mock_iam()
+def test_list_roles_with_description():
```

Comment:
The logic seems to be the same as `test_list_roles_with_empty_string_description` - would you mind parameterizing this method to cover both cases? IMO that would be easier to maintain
```
@pytest.mark.parametrize("desc", ["", "Test Description"])
def test_list_roles_with_description(desc):
```

Reply:
Thanks - updated.